deps.edn

```diff
@@ -0,0 +1,4 @@
+{:paths ["src" "resources"],
+ :deps
+ {mvxcvi/alphabase {:mvn/version "2.1.0"},
+  commons-codec/commons-codec {:mvn/version "1.14"}}}
```

Comment:
:thinking: A bit of a tangent to this PR, but given your change to the `b16` namespace, I'm curious what you think about fully removing the commons-codec dependency and just using the native clojure implementations for all the base encoding needs. I'm sure the Java implementations are more efficient, but removing this dependency is appealing.

Reply:
>  I'm curious what you think about fully removing the commons-codec dependency

I'd be in favor of that from the bb perspective